src/core/lib/gprpp/validation_errors.cc

```diff
@@ -42,8 +43,13 @@ bool ValidationErrors::FieldHasErrors() const {
   return field_errors_.find(absl::StrJoin(fields_, "")) != field_errors_.end();
 }
 
-absl::Status ValidationErrors::status(absl::string_view prefix) const {
+absl::Status ValidationErrors::status(absl::string_view prefix,
```

Comment:
Can we add `absl::StatusCode` as the first argument? (to mirror the `absl::Status` constructor)

Reply:
I agree, it would probably make sense to make this a mandatory parameter and put it first.  It shouldn't be hard to update the callers.

Reply:
Good idea. Done.